ls1intum/Artemis
# Allow fullscreen lecture units for more browsers

### Checklist

### Motivation and Context



fixes #3267

### Description
strict feature policy header was blocking fullscreen iframes for some chromium based browsers. Honestly seems like a bug to me because the iframe should be able to override this but loosening the feature policy does the trick.

### Steps for Testing


0. Find a browser that doesn't allow full screen lecture units on the current Artemis instance (Chrome should be a good fit. In my case it's Chromium v 90.0.4430.85 (snap)). If you can't find a browser that doesn't work you won't be able to review this :(
1. Create a video unit (try a youtube video and a TUM-Live video, e.g. `https://live.mm.rbg.tum.de/w/eist/991?video_only=1`)
2. Check if fullscreen works now.
3. Check if fullscreen still works in other browsers.
